CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# [1.1.0](https://github.com/CESARBR/knot-fog-connector-knot-cloud/compare/v1.0.2...v1.1.0)
+
+### Features
+
+- Handle disconnection/reconnection.
+
+### Bug Fixes
+
+- Fix return event listeners.
+
+## [1.0.2](https://github.com/CESARBR/knot-fog-connector-knot-cloud/compare/v1.0.1...v1.0.2)
+
+### Bug Fixes
+
+- Listen to commands right after a device is registered.
+
+## [1.0.1](https://github.com/CESARBR/knot-fog-connector-knot-cloud/compare/v1.0.0...v1.0.1)
+
+### Improvements
+
+- Update knot-cloud-websocket to v1.0.1.
+
+# [1.0.0](https://github.com/CESARBR/knot-fog-connector-knot-cloud/compare/v0.1.0...v1.0.0)
+
+### Bug Fixes
+
+- Add `knot:` prefix to types.
+- Fix device format returned by listDevice().
+
```

Comment:
[v0.1.1](https://github.com/CESARBR/knot-fog-connector-knot-cloud/releases/tag/v0.1.1) isn't described here.

Reply:
I didn't describe it because it is the same as v1.0.0 and it was likely an error in setting the version. See that from v0.1.1 to v1.0.0 there isn't any changes besides setting the new version, both commits are from the same day and there are changes that are more than a backwards compatible bug fix (e.g. "Return devices as expected by the new API" and "Use 'knot:' prefix to type names"). I understand that the real version update should be to change to v1.0.0 (and we can't erase it because it was published to NPM).

Would you write about it here in some way?

Reply:
Ack. I agree with you about describing only the v1.0.0. I wouldn't add nothing more, so we can integrate this PR.